workload-metrics/manifests/workload-metrics-podmon.yaml

```diff
@@ -0,0 +1,31 @@
+# Note that this PodMonitor is in the monitoring.gke.io domain,
+# rather than the monitoring.coreos.com domain used with the
+# Prometheus Operator
+apiVersion: monitoring.gke.io/v1alpha1
+kind: PodMonitor
```

Comment:
Where is the documentation for this resource? 
Is this feature launched?

Reply:
We have a chicken and egg problem ... the code is available but the blog and docs have not yet been released.